packages/modules/qdrant/src/qdrant-container.ts

```diff
@@ -0,0 +1,73 @@
+import { AbstractStartedContainer, GenericContainer, StartedTestContainer, Wait } from "testcontainers";
+
+const QDRANT_REST_PORT = 6333;
+const QDRANT_GRPC_PORT = 6334;
+const QDRANT_CONFIG_FILE_PATH = "/qdrant/config/config.yaml";
+
+export class QdrantContainer extends GenericContainer {
+  private apiKey: string | undefined;
+  private configFilePath: string | undefined;
+
+  constructor(image = "qdrant/qdrant:latest") {
+    super(image);
+    this.withExposedPorts(QDRANT_REST_PORT, QDRANT_GRPC_PORT);
+    this.withWaitStrategy(Wait.forLogMessage(/Actix runtime found; starting in Actix runtime/));
+    this.withWaitStrategy(Wait.forHttp("/readyz", QDRANT_REST_PORT));
```

Comment:
`withWaitStrategy` will _replace_ an existing wait strategy. If you want multiple wait strategies you can use the CompositeWaitStategy, e.g:

```js
.withWaitStrategy(Wait.forAll([
  Wait.forLogMessage(/Actix runtime found; starting in Actix runtime/),
  Wait.forHttp("/readyz", QDRANT_REST_PORT)
]))
```

Reply:
https://github.com/testcontainers/testcontainers-node/pull/725/commits/723d9750c253f840ec536d81a7c2eeb0b8667505